index.html

```diff
@@ -0,0 +1,112 @@
+                <h3>2025년7월3일 ~ 8월2일</h3>
+                <p class="venue">@SEOUL Olympic Stadium, TOKYO Dome, Prince Moulay Abdellah Stadium and more ~ </p>
+            </div>
+        </section> 
+
+        <section class="schedule">
+            <h2>Main program</h3>
+            <span class="line-break"></span>
+            <div class="program-container">
+                <div class="program">
+                  <img class="img1" src="./assets/img/program3.png" alt="">
+                  <h3>Press Conference</h3>
+                  <p>The members will answer on press questions and talk about their journey as solo artists before army enlistment</p>
+                </div>
+                <div class="program">
+                  <img class="img2" src="./assets/img/Program2.png" alt="">
+                  <h3>Concert Day 1</h3>
+                  <p>Voice check starts at 9 am. The concert takes place from 7 to 10pm. Songs from HYYH, Wings and Love Yourself alumbs will be performed</p>
+                </div>
+                <div class="program">
+                  <img class="img2" src="./assets/img/Program2.png" alt="">
+                  <h3>Concerts Day 2</h3>
+                  <p>Voice check starts at 9 am. The concert takes place from 7 to 10pm. Songs from Map of The Soul, Proof and solo albums will be performed</p>
+                </div>
+                <div class="program">
+                  <img class="img2" src="./assets/img/Program5.png" alt="">
+                  <h3>Fan Meeting</h3>
+                  <p>The members will join their fans in an exlusive meeting to sign albums and have fun together with Armys</p>
+                </div>
+                <div class="program">
+                  <img class="img1" src="./assets/img/Program1.png" alt="">
+                  <h3>Show Time</h3>
+                  <p>Show of BTS three movies: Burn The Stage, Bring The Soul and Break The Silence</p>
+                </div>
+              </div>
+              <a href="#">Get your tickets now on Weverse Shop</a>
+        </section>
+        <!------------------ GROUP MEMEBERS SECTION ------------------>
+        <section class="global-section">
+            <h3>Group Members</h3>
+            <hr>
+            <section class="gm-section">
+                <!--- Dynamically generated with javascript --->
+            </section>
+        </section>  
+        <!------------------ PARTNERS SECTION ------------------>
+        <section class="partners">
+            <h3 class="partners-title">Partners</h3>
+            <hr>
+            <ul>
+              <li><img src="./assets/img/hybe.png" alt=""></li>
```

Comment:
- [ ] I noticed that some of your images have an empty **alt** attribute, please make sure to fix it The alt attribute provides alternative information for an image if a user for some reason cannot view it (because of a slow connection, an error in the src attribute, or if the user uses a screen reader). 👍